src/EditorFeatures/CSharpTest2/Recommendations/NullKeywordRecommenderTests.cs

```diff
@@ -308,5 +308,19 @@ public async Task TestAfterIs_BeforeExpression()
 int y = x is $$ Method();
 "));
         }
+
+        [Fact]
+        public async Task TestAfterLambdaOpenParen()
+        {
+            await VerifyKeywordAsync(
+@"var lam = ($$");
+        }
+
+        [Fact]
+        public async Task TestAfterLambdaComma()
+        {
+            await VerifyKeywordAsync(
+@"var lam = (int i, $$");
```

Comment:
we have an equivalent params test for this case, right?

Reply:
Yes, it's already in the feature branch:

https://github.com/dotnet/roslyn/blob/b2325a6242c6beec480b23e9eee603b5db310cfb/src/EditorFeatures/CSharpTest2/Recommendations/ParamsKeywordRecommenderTests.cs#L223-L228